lib/msal-core/samples/quickStart/Readme.txt

```diff
@@ -0,0 +1,15 @@
+
+How to run the dev apps:
+--------------------
+Pre-requisite
+- Install node.js if needed (https://nodejs.org/en/)
+
+Resolving the server.js references
+- In a command prompt, run npm install
+
+Running the sample
+- In a command prompt, run �node server.js�
```

Comment:
I like that we are supporting Node server for non Windows users.
However, we do want to have a Visual studio solution as well for Windows users as structured in the existing Quickstart:
https://github.com/Azure-Samples/active-directory-javascript-graphapi-v2
I believe even Andre's PR when we reviewed it had the VS and the Node server option.

Reply:
This project can be used with visual studio code without any changes. Infact it can be opened with other popular IDE as well like eclipse, intellij etc.  I haven't tried it with visual studio solution. Is that a requirement? For Javascript/typescript people tend to use visual studio code more than visual studio solution.